api/v1/storageautoscaler_types.go

```diff
@@ -122,7 +122,8 @@ type StorageAutoScalerStatus struct {
 // +kubebuilder:printcolumn:name="Age",type=date,JSONPath=.metadata.creationTimestamp, description="Age"
 // +kubebuilder:printcolumn:name="Phase",type=string,JSONPath=.status.phase,description="Current Phase"
 // +kubebuilder:printcolumn:name="DeviceClass",type=string,JSONPath=.spec.deviceClass,description="Device Class"
-// +kubebuilder:printcolumn:name="LastRunTimeStamp",type=date,JSONPath=.status.lastRunTimeStamp,description="Last Run Time Stamp"
+// +kubebuilder:printcolumn:name="LastRunStartTimeStamp",type=date,JSONPath=.status.lastExpansion.startTime,description="Last Run Start Time Stamp"
```

Comment:
nit: "time" instead of "timestamp"
```suggestion
// +kubebuilder:printcolumn:name="LastRunStartTime",type=date,JSONPath=.status.lastExpansion.startTime,description="Last Run Start Time"
```